docs/Create-Packages/Creating-a-Package.md

```diff
@@ -385,6 +388,28 @@ With NuGet 3.x, targets are not added to the project but are instead made availa
 
 <a href="creating-the-package"></a>
 
+## Authoring COM Interop packages compatible with Package Reference
+
+In the packages.config world, when adding references to the assemblies from the packages NuGet and Visual Studio would test which assemblies are interop and set the EmbedInteropTypes to true.
+
+In the Package Reference case, the EmbedInteropTypes metadata is always false for all assemblies. We require package authors to explicitly add this metadata by including a [targets file](#including-msbuild-props-and-targets-in-a-package).
+ It's very important that you make this target name as unique as possible to avoid clashes. Ideally you would include both your package name and the assembly being embedded in the target name.
+See sample project [here](https://github.com/NuGet/Samples/tree/master/NuGet.Samples.Interop).
+
+```xml
+      
+ <Target Name="EmbeddingAssemblyNameFromPackageId" AfterTargets="ResolveReferences" BeforeTargets="FindReferenceAssembliesForReferences">
+   <PropertyGroup>
+     <_InteropAssemblyFileName>NuGet.Samples.Interop</_InteropAssemblyFileName>
```

Comment:
Instead of actually having a sample name there, should we have something {InteropAssemblyName}?